tests/test_sanity_check.py

```diff
@@ -35,19 +100,32 @@ def run_command(command: List[str], **kwargs: Any) -> None:
                 run_command(version_command, env=env_with_pantsd)
                 run_command(list_command, env=env_with_pantsd)
 
-    def check_for_all_python_versions(
-        self, *python_versions: str, pants_version: Optional[str], use_toml: bool = True
-    ) -> None:
-        for python_version in python_versions:
-            self.sanity_check(
-                pants_version=pants_version, python_version=python_version, use_toml=use_toml
-            )
 
-    def test_pants_latest_stable(self) -> None:
-        self.sanity_check(python_version=None, pants_version=None, use_toml=False)
-        self.check_for_all_python_versions("3.6", "3.7", "3.8", pants_version=None, use_toml=False)
+@pytest.fixture
+def sanity_checker(pyenv_bin: str, pyenv_versions: List[str], build_root: Path) -> SanityChecker:
+    return SanityChecker(pyenv_bin=pyenv_bin, pyenv_versions=pyenv_versions, build_root=build_root)
+
+
+def check_for_all_python_versions(
+    sanity_checker: SanityChecker,
+    *python_versions: str,
+    pants_version: Optional[str],
+    use_toml: bool = True,
+) -> None:
+    for python_version in python_versions:
+        sanity_checker.sanity_check(
+            pants_version=pants_version, python_version=python_version, use_toml=use_toml
```

Comment:
Consider moving this to be a method on `SanityChecker`. Then the test would read:
```python
sanity_checker.sanity_check(python_version=None, pants_version="1.26.0.dev0")
sanity_checker.check_for_all_python_versions("3.6", "3.7", "3.8", pants_version="1.26.0.dev0")
```

Reply:
Done.